### ISSUE TITLE
ShuffleNetV1 architecture differ from paper. Some pretrained weights are corrupted

### ISSUE THREAD

**Issue by vlad3996** [megvii-model/ShuffleNet-Series#4]
According to the [paper](https://arxiv.org/pdf/1707.01083.pdf) output channels for _**group=8**_  are` [-1, 24, 384, 768, 1536]`, but you are using `[-1, 24, 240, 480, 960]` for group=3 and group=8 [here](https://github.com/megvii-model/ShuffleNet-Series/blob/master/ShuffleNetV1/network.py#L16) .

Some state dicts are corrupted (containing mainly -inf and nan values) :

- group3 0.5x

- group8 {0.5x, 1.0x, 1.5x, 2.0x}

**Comment by nmaac** [comment]
Hi @vlad3996 , thanks for reporting.

1. Same as in the paper, we trained ShuffleNetV1(g=3) with channels [-1, 24, 240, 480, 960], and trained ShuffleNetV1(g=8) with channels [-1, 24, 384, 768, 1536]. But the code in this repo only contains channel settings using group=3, now we have added the case using group=8.

2. For those corrupted models we re-uploaded them, please check thanks.

**Comment by vlad3996** [comment]
@nmaac thanks!
Almost everything works fine. I think that you upload `group8 2.0x` instead of `group3 0.5x`.

**Comment by nmaac** [comment]
Fixed.

**Comment by nmaac** [comment]
> @nmaac thanks!
> Almost everything works fine. I think that you upload `group8 2.0x` instead of `group3 0.5x`.

Fixed.
